src/Runner/DefaultTestResultCache.php

```diff
@@ -62,11 +62,11 @@ final class DefaultTestResultCache implements \Serializable, TestResultCache
      * $this->times[$testName] = 1.234;
      * </code>
      *
-     * @var array<string, float>
+     * @var float[]
```

Comment:
This annotation is also correct (and both save you having to use `@var` docblocks above `foreach` loops)

Reply:
Done.